.ci/deps.sh

```diff
@@ -58,7 +58,7 @@ if [ ! -e ~/phpmd/phpmd ]; then
 fi
 
 # astyle installation
-wget "https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia" -O ~/astyle.tar.gz
+wget "https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia" -O ~/astyle.tar.gz --no-check-certificate
```

Comment:
after upgrading ca-certificates,
--no-check-certificate will no more required

Reply:
Definitely a better idea to update the ca-certs if they are not being updated by the image you have choosen.

Reply:
Cool😉